### ISSUE TITLE
Where?

### ISSUE THREAD

**Issue by Berserk0001** [Issaminu/bandwidth-hero-proxy#1]
I cannot see the commit where you have solved the issue..

**Comment by Issaminu** [comment]
The project bio was a hint :)

![image](https://github.com/user-attachments/assets/dbe72020-ad2a-4ee2-9d0c-b8ac3213d19c)

If you look at the commit history, you'll find thousands of `Append 'a' to file.txt` commits, that's a [Github Action scheduled to commit to a file every 30 mins](https://github.com/Issaminu/bandwidth-hero-proxy/blob/main/.github/workflows/append.yml), which would trigger my hosting provider (Railway) to redeploy the application.

So basically, I force a redeployement on CI by having a cron job make a commit every 30 mins, which then "resets" the app's memory usage.

This is obviously a cursed solution, but it works so I'm fine with that  ¯\_(ツ)_/¯
Anyways, you can do this in a much simpler way by just having the application restart every X minutes if your way of hosting the application allows you to do that , that would effectively negate the effect of the memory leak.

**Comment by Issaminu** [comment]
Closing for inactivity.
